### ISSUE TITLE
[IOS-XE]  SchemaMissingKeyError:  Show Lisp Service Commands

### ISSUE THREAD

**Issue by jobower-at-cisco** [CiscoTestAutomation/genieparser#912]
Getting the following SchemaMissingKeyError:

```
2024-11-04 17:28:12,627: %UNICON-INFO: +++ svs-fc-agg-t2-a-3 with via 'ssh': executing command 'show lisp all extranet ext1 instance-id 1' +++
show lisp all extranet ext1 instance-id 1

================================================
Output for router lisp 0
================================================
% Extranet table ext1 not found.
svs-fc-agg-t2-a-3#	
12:28:12.906	FAIL	SchemaMissingKeyError: Show Command: show lisp all extranet ext1 instance-id 1
Missing keys: [['lisp_id', 0, 'total'], ['lisp_id', 0, 'home_instance']] 
```

Command is being run through the LISP Profile Model for IOSXE/cat9k, currently making the model unusable in our tests.
Thanks for the help!
John

**Comment by jobower-at-cisco** [comment]
> Hello @jobower-at-cisco,
> 
> I have found the fix for this issue and now the output looks like this:
> 
> ```
> {
>     "lisp_id": {
>         0: {}
>     }
> }
> ```
> 
> I have raised the PR for this and once it is merged, I will update you.
> 
> Thank you.

Thanks @SohanTirpude

**Comment by SohanTirpude** [comment]
Hello @jobower-at-cisco,

The PR just got merged and the fix will be available in the upcoming pyats 24.11 release. Hence, I am closing this ticket for now. If you are facing any other issues, kindly open a new ticket.

Thank you.
